diff --git a/node_modules/react-languages-select/lib/languages.js b/node_modules/react-languages-select/lib/languages.js
index 8b29a63..6ea39a0 100644
--- a/node_modules/react-languages-select/lib/languages.js
+++ b/node_modules/react-languages-select/lib/languages.js
@@ -3,6 +3,7 @@
 exports.__esModule = true;
 exports.languagesX = exports.languages = void 0;
 var languages = {
+  "ua": "Українська",
   "sq": "shqiptar",
   "ar": "العربية",
   "bn": "বাংলা",
@@ -30,6 +31,11 @@ var languages = {
 };
 exports.languages = languages;
 var languagesX = {
+  "ua": {
+    "name": "Ukraine",
+    "localName": "Українська",
+    "countries": ["Ukraine"]
+  },
   "sq": {
     "name": "Albanian",
     "localName": "shqiptar",
